tests/platform/document/document.spec.ts

```diff
@@ -92,7 +92,7 @@ test('document', async ({ page }) => {
   await expect(testCreateElementError_).toHaveText('no error');
 
   const testCreateHTMLDocument = page.locator('#testCreateHTMLDocument');
-  await expect(testCreateHTMLDocument).toHaveText('88mph hidden');
+  await expect(testCreateHTMLDocument).toHaveText('88mph hidden object');
```

Comment:
@adamdbradley this is where I added the test from [#174 Comment](https://github.com/BuilderIO/partytown/issues/174#issuecomment-1121708179)